Alignment/TrackerAlignment/plugins/AlignmentPrescaler.cc

```diff
@@ -148,17 +187,18 @@ void AlignmentPrescaler::produce(edm::Event& iEvent, const edm::EventSetup& iSet
             tmpflag.SetDetId(hit->geographicalId());
             if (tmpflag.isOverlap())
               isOverlapHit = true;
-            // std::cout<<"~*~*~* Prescale (2D) for module "<<tmpflag.detId().rawId()<<"("<<InValMap[stripclust].detId().rawId() <<") is "<<hitPrescFactor_<<std::flush;
-            //  if(tmpflag.isOverlap())cout<<" (it is Overlap)"<<endl;
-            // else cout<<endl;
-
+            LogDebug("AlignmentPrescaler")
+                << "~*~*~* Prescale (2D) for module " << tmpflag.detId().rawId() << "("
+                << InValMap[stripclust].detId().rawId() << ") is " << hitPrescFactor_ << std::flush;
```

Comment:
I think the std::flush is not needed, the edm::LogPrint will take care of it, no?